tests/unittests/Foundation/NSStringTests.m

```diff
@@ -727,10 +727,15 @@ void testUrlCharacterSetEncoding(NSString* decodedString, NSString* encodedStrin
 }
 
 TEST(NSString, ContainsString) {
+    EXPECT_FALSE([@"" containsString:nil]);
     EXPECT_FALSE([@"" containsString:@""]);
     EXPECT_FALSE([@"TÉST" containsString:@""]);
     EXPECT_TRUE([@"TÉST" containsString:@"T"]);
     EXPECT_TRUE([@"TÉST" containsString:@"TÉST"]);
+    EXPECT_FALSE([@"TE\u0301ST" containsString:@"TÉST"]);
+    EXPECT_FALSE([@"TÉST" containsString:@"TE\u0301ST"]);
+    EXPECT_TRUE([@"اختبار النص" containsString:@"اختبار النص"]);
```

Comment:
I don't think we should be testing containsString's ability to translate #ByDesign